processing/python/display_python/display_python.pyde

```diff
@@ -0,0 +1,55 @@
+from surfaces import *
+import getpass
+
+add_library('oscP5') 
+addr = "?"
+
+class Listen(OscEventListener):
+    def oscEvent(self, m):
+        global loc, osc
+        if m.checkAddrPattern("/beat")==True:
+            current_beat = "".join([str(i) for i in list(m.arguments()[0])][0])
+            change_color = "".join([str(i) for i in list(m.arguments()[1])])
+            current_part_name = "".join([str(i) for i in list(m.arguments()[2])])
+            next_part_name = "".join([str(i) for i in list(m.arguments()[3])])
+            AREAS['part_info'].update_parts(current_part_name, next_part_name, current_beat, change_color)
+        elif m.checkAddrPattern("/display_input") == True:
+            utterance = "".join([str(i) for i in list(m.arguments()[0])])
+            category = "".join([str(i) for i in list(m.arguments()[1])])
+            AREAS['utterances'].update_utts(utterance, category)
+
+        
+
+def setup():
+    size(1200, 850)
+    background(200)
+    global font, font2, font_size
+    font_size = 14
+    font = createFont("Arial-BoldMT", font_size, True)
+    font2 = createFont( "Helvetica Neue", font_size, True)
+    global osc, loc
+    AREAS = build_areas2(font2)
```

Comment:
Since the original `build_areas` is not in use, it would be nice to remove the `2` from the name (same for `font`)